pkg/utils/utils.go

```diff
@@ -413,6 +413,11 @@ func IsServiceNSValid(namespace string) bool {
 	return true
 }
 
+func IsVCFCluster() bool {
+	vcfCluster := os.Getenv(VCF_CLUSTER)
+	return vcfCluster == "true"
```

Comment:
@monotosh-avi not as a part of this PR, but can we modify this function to take in the `parseBool` to validate the input instead of a string match?

Reply:
done